shared/cryptoutil/cryptoUtil.ts

```diff
@@ -190,3 +196,9 @@ export function hmacSHA(encryptionKey: string, data: string) {
   const resultBytes = hmac.digest().getBytes().toString();
   return resultBytes;
 }
+
+export async function compressData(fileName: string): Promise<string> {
```

Comment:
Let's move this to fileStorage as there is not cryptographic involved here.
It's simple file zip.

Reply:
Okay.